msbuild/Xamarin.Mac.Tasks/Tasks/Metal.cs

```diff
@@ -1,13 +1,18 @@
 ﻿using System.IO;
 
+using Xamarin.MacDev;
 using Xamarin.MacDev.Tasks;
 
 namespace Xamarin.Mac.Tasks
 {
 	public class Metal : MetalTaskBase
 	{
 		protected override string OperatingSystem {
-			get { return "osx"; }
+			get { return "macosx"; }
+		}
+
+		protected override string MinimumDeploymentTargetKey {
+			get { return ManifestKeys.LSMinimumSystemVersion; }
```

Comment:
not a new problem but I thought the leaf classes where supposed to be empty so XVS could replace them (for remoting) ?

Reply:
Yea, and for iOS we do that, but I think the Mac tasks were done before XVS started supporting that.

It's probably something we'll need to refactor into the same structure we have for iOS.